app/models/work.rb

```diff
@@ -0,0 +1,25 @@
+class Work < ApplicationRecord
+  validates :title, presence: true, uniqueness: true
+  validates :category, presence: true
+
+  has_many :votes
+  has_many :users, :through => :votes
+
+  def self.most_voted
+    max_votes = 0
+    result = nil
+    Work.all.each do |work|
+      num_votes = work.vote_ids.length
+      if num_votes > max_votes
+        max_votes = num_votes
+        result = work
+      end
+    end
```

Comment:
Hm, is there a way we could refactor and use `max_by`?
Consider the following refactor, which passes all the tests:
```ruby
  def self.most_voted
    result = Work.all.max_by do |work|
      work.votes.length
    end
    return result
  end
```